components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/OperationPolicySpecAttribute.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+package org.wso2.carbon.apimgt.api.model;
+
+import java.util.Objects;
+
+public class OperationPolicySpecAttribute {
+
+    private String name;
+    private String displayName;
+    private String description;
+    private String validationRegex;
+    private String type;
+    private boolean required;
```

Comment:
don't we need defaultValue

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/11040/commits/aabf4548185a3c63b94e1943085e1f4d99363450